build/NugetConfigFile.targets

```diff
@@ -28,9 +28,8 @@
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="vstest" value="https://dotnet.myget.org/F/vstest/api/v3/index.json" />
-    <add key="web-api" value="https://dotnet.myget.org/F/dotnet-web/api/v3/index.json" />
-    <add key="symreader-native" value="https://dotnet.myget.org/F/symreader-native/api/v3/index.json" />
-    <add key="nuget-build" value="https://dotnet.myget.org/F/nuget-build/api/v3/index.json" /> 
+    <add key="nuget-build" value="https://dotnet.myget.org/F/nuget-build/api/v3/index.json" />
+    <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" /> 
```

Comment:
The addition of the 'dotnet-corefxlab' feed was a fix for specific ASP.NETcore.App versions in the orchestrated build for release/2.1.3xx that took dependencies on non-published packages: System.IO.Pipelines 0.1.0-preview1-180129-1 and System.Text.Encodings.Web.Utf8 0.1.0-preview1-180129-1 - That specific line may not be necessary in CLI:master.